src/views/common.tsx

```diff
@@ -1,4 +1,6 @@
-import { Html } from '@kitajs/html'
+import type { PropsWithChildren } from '@kitajs/html'
+import { escapeHtml } from '@kitajs/html'
```

Comment:
why the change in import? It's now generating warnings

Reply:
Because it's no longer under `{ Html }` and also mentioned in https://github.com/kitajs/docs/pull/36/files#diff-4e4ca54733d4a35f3b755176b22a0cca067a01ad2e9b28f94c1881544a221c0f deprecations so was just following their suggestions. The `import { Html }` generates warning as per screenshot below:
<img width="534" alt="Screenshot 2024-07-31 at 13 07 46" src="https://github.com/user-attachments/assets/fef45864-ffda-493f-9c98-fbdd3e259b37">

So imported PropsWithChildren as a type and escapeHtml as it's, open to suggestions.

Reply:
<img width="830" alt="Screenshot 2024-07-31 at 11 13 54" src="https://github.com/user-attachments/assets/f7e1dddf-ca88-4478-b6d3-8e2135f5eae6">

Reply:
this should be reflected in the commit.